examples/translator/client/translator.go

```diff
@@ -17,30 +17,71 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 
 	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/status"
 
 	translator_pb "github.com/project-oak/oak/examples/translator/proto"
+	label_pb "github.com/project-oak/oak/oak/proto/label"
 )
 
 var (
 	address = flag.String("address", "localhost:8080", "Address of the Oak application to connect to")
 	caCert  = flag.String("ca_cert", "", "Path to the PEM-encoded CA root certificate")
 )
 
+// Keep in sync with /oak/server/rust/oak_runtime/src/node/grpc/server/mod.rs.
+const oakLabelGrpcMetadataKey = "x-oak-label-bin"
```

Comment:
This seems like it should (eventually) live in a Go `oak_client` package, together with the logic for creating metadata from an actual label. Perhaps create an issue + TODO?

Reply:
Good point, added #1097